components/inputs/input-checkbox.js

```diff
@@ -205,11 +208,6 @@ class InputCheckbox extends SkeletonMixin(RtlMixin(LitElement)) {
 		`;
 	}
 
-	focus() {
-		const elem = this.shadowRoot && this.shadowRoot.querySelector('input.d2l-input-checkbox');
-		if (elem) elem.focus();
-	}
```

Comment:
Pretty much all of these look like this. There were a couple components doing complex focus logic (like `<d2l-calendar>`) and I just left them as they were.